papermill/execute.py

```diff
@@ -388,7 +388,14 @@ def raise_for_execution_errors(nb, output_path):
     if error:
         # Write notebook back out with the Error Message at the top of the Notebook.
         error_msg = ERROR_MESSAGE_TEMPLATE % str(error.exec_count)
-        error_msg_cell = nbformat.v4.new_markdown_cell(source=error_msg)
+        error_msg_cell = nbformat.v4.new_code_cell(
+            source="%%html\n" + ERROR_MESSAGE_TEMPLATE,
+            outputs=[
+                nbformat.v4.new_output(
+                    output_type="display_data",
+                    data={"text/html": ERROR_MESSAGE_TEMPLATE}
+                )]
+        )
```

Comment:
Is this going to hide the input?

Reply:
I still need to stick some metadata on to hide the input.

Reply:
Updated this to use the metadata recognized by multiple frontends for hidden cells.